contrib/k8s/README.md

```diff
@@ -27,17 +26,14 @@ dex needs a secret key for encrypting private keys in the database. These can be
 
 [k8s-secrets]: http://kubernetes.io/v1.0/docs/user-guide/secrets.html
 
-```
-kubectl create -f dex-secrets.yaml
-```
+The secret will be created as part of the next command, from `dex-overlord.yaml`
```

Comment:
Would it be possible to add an example here of creating the secret? For example:

```
dd if=/dev/random bs=1 count=32 2>/dev/null | base64 | tr -d '\n' > key-secrets
kubectl create secret generic dex --from-file=./key-secrets
```